db/queries/assessment_records/helpers.py

```diff
@@ -0,0 +1,48 @@
+from sqlalchemy import select, cast, func
+from sqlalchemy.dialects.postgresql import JSONB
+import json
+from db import db
+
+COF_json_mapper = {
+    # "column name" : "jsonpath to value in jsonb blob".
+    "application_id": "$.id",
+    "project_name": "$.project_name",
+    "short_id": "$.reference",
+    "fund_id": "$.fund_id",
+    "round_id": "$.round_id",
+    "funding_amount_requested": (
+        '$.forms[*].questions[*].fields[*] ? (@.key == "JzWvhj")."answer"'
+    ),
```

Comment:
Not sure if this is better pulled through to the top level in an import script?

Also I think the intention is for this to be capital + revenue  so `JzWvhj` + `jLIgoi`

Reply:
Thank you! 😃